var/spack/repos/builtin/packages/beatnik/package.py

```diff
@@ -47,13 +48,14 @@ class Beatnik(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("kokkos +wrapper", when="%gcc+cuda")
 
     # Cabana dependencies
-    depends_on("cabana @0.6.0 +grid +heffte +silo +hdf5 +mpi")
+    depends_on("cabana @master +grid +heffte +silo +hdf5 +mpi +arborx")
```

Comment:
Note that this will apply to all versions of this package. Is it correct for v1.0, given that previously the version was 0.6.0?

Also, it doesn't seem great that a released version of something depends on an unspecified development version of a dependency. The recipe may work now, but I guess won't work as soon as `cabana@master` introduces some breaking changes. Can we fix that too?

Reply:
I doubled-checked v1.0 with cabana@0.7.0 and it builds and runs without errors. So yes, cabana@0.7.0 is correct for v1.0.

The spec has been updated to depend on cabana@0.7.0 for all versions minus develop, which depends on cabana@master.